### DISCUSSION TITLE
Can I send polls out as a message?

### DISCUSSION THREAD

**Discussion by philippcheung** [advanced-chat/advanced-chat-components#437] [General]
Hello,

My apologies if this is already do-able, but I'm wondering if I can use vue-advanced-chat with not just real users, but also with a support channel.  In this case, i'd like to ask a few screening questions and then connect the person to the right chat user.  

So upon starting a chat with "support", the first question might be -- How can we help you?  1) I have a billing question, 2) I have a support question, 3) other.

Maybe not so much polling, but fixed responses, rather than free form responses.

**Comment by antoine92190** [comment]
You can do whatever you want, this library is just a UI component. You can use the `messages` prop to fill in the chat messages how you see fit, such as sending messages from the interlocutor to fake a real person.
